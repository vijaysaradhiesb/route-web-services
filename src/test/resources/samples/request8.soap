<soapenv:Envelope xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:soapenv="http://schemas.xmlsoap.org/soap/envelope/" xmlns:ws="http://ws.integclaims.itf.com">
   <soapenv:Header/>
   <soapenv:Body>
        <ws:FindRefData soapenv:encodingStyle="http://schemas.xmlsoap.org/soap/encoding/">
            <username xsi:type="xsd:string">webservice</username>
            <password xsi:type="xsd:string">password</password>
            <TABLE_NAME xsi:type="xsd:string">RD_TITLE</TABLE_NAME>
        </ws:FindRefData>
   </soapenv:Body>
</soapenv:Envelope>