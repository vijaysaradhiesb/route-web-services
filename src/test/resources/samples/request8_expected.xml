<?xml version="1.0" encoding="UTF-8" standalone="yes"?><ns2:GenericWebServiceMessage xmlns="http://ws.integclaims.itf.com" xmlns:ns2="http://generic.message.integclaims.integ.com"><ns2:name>FindRefData</ns2:name><ns2:keyValuePair><ns2:key>username</ns2:key><ns2:valueType>STRING</ns2:valueType><ns2:stringValue>webservice</ns2:stringValue></ns2:keyValuePair><ns2:keyValuePair><ns2:key>password</ns2:key><ns2:valueType>STRING</ns2:valueType><ns2:stringValue>password</ns2:stringValue></ns2:keyValuePair><ns2:keyValuePair><ns2:key>TABLE_NAME</ns2:key><ns2:valueType>STRING</ns2:valueType><ns2:stringValue>RD_TITLE</ns2:stringValue></ns2:keyValuePair></ns2:GenericWebServiceMessage>